common/p_horde.cpp

```diff
@@ -1070,3 +1150,59 @@ BEGIN_COMMAND(hordeinfo)
 	Printf("Boss Damage: %d\n", ::g_HordeDirector.serialize().bossDamage);
 }
 END_COMMAND(hordeinfo)
+
+BEGIN_COMMAND(hordecooldownclear)
+{
+	if (!G_IsHordeMode())
+	{
+		Printf("Can't clear the horde cooldown list outside of horde mode.\n");
+		return;
+	}
+
+	if (network_game && clientside)
+	{
+		Printf(serveronlycmd);
+		return;
+	}
+
+	for (int clearlist = 0; clearlist < HORDECOOLDOWN_SIZE; clearlist++)
+	{
+		hordecooldown[clearlist] = NULL;
+	}
+	hordecoolcount = 0;
+}
+END_COMMAND(hordecooldownclear)
+
+BEGIN_COMMAND(hordecooldownprint)
+{
+	if (!G_IsHordeMode())
+	{
+		Printf("Can't obtain horde cooldown list outside of horde mode.\n");
+		return;
+	}
+
+	if (network_game && clientside)
+	{
+		Printf(serveronlycmd);
```

Comment:
Needs a string literal like "%s\n" or it won't compile on linux

Reply:
Done. The compulsory new line is placed at the end of the string variable.